pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -340,12 +340,10 @@ func filterTerminatedContainerInfoAndAssembleByPodCgroupKey(containerInfo map[st
 	}
 	result := make(map[string]cadvisorapiv2.ContainerInfo)
 	for _, refs := range cinfoMap {
-		if len(refs) == 1 {
```

Comment:
Changed the original behavior. Need to consider whether other scenarios are also ok

Reply:
If only one container instance, it still may be an exited instance(for init container), so we should not process len() == 1 specially